src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,33 @@
+import clsx from 'clsx'
+import React, { useEffect, useRef, useState } from 'react'
+
+interface Props {
+  text: string
+  className?: string
+  children: React.ReactNode
+}
+
+const Tooltip: React.FC<Props> = ({ text, children, className }) => {
+  const [height, setHeight] = useState(0)
+  const parentWrapperRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (parentWrapperRef.current) {
+      setHeight(parentWrapperRef.current.clientHeight)
+    }
+  })
+
+  return (
+    <div className={clsx('has-tooltip', className)} ref={parentWrapperRef}>
```

Comment:
Ich versuche ja Dinge in den .css files zu vermeiden, sondern die Dinge schön contained zu halten.

```suggestion
    <div className={clsx('has-tooltip', className)} ref={parentWrapperRef}>
```

Ich glaube fast, in diesem Fall könnte ein `useState` es gut lesbar machen.

Du kannst es aber IMO auch vollständig mit TW machen:
https://play.tailwindcss.com/MJHzCxpIQN

```
<div class="group/tooltip cursor-help border border-red-200">
  <span class="absolute top-2 hidden cursor-default select-none hyphens-none break-keep rounded border bg-gray-900 p-2 text-white shadow-md group-hover/tooltip:z-50 group-hover/tooltip:block">Foo</span>

  Lorem Ipsum
</div>

```

Reply:
Ok, danke. Das muss ich mir nochmal genauer anschauen :)

Reply:
Das mit `useState` habe ich nicht verstanden. Wo sollte ich das benötigen?